src/app/screens/BookDetail/components/BookDetail/styles.js

```diff
@@ -0,0 +1,71 @@
+import { StyleSheet } from 'react-native';
+import colors from '../../../../../constants/colors';
+
+export default StyleSheet.create({
+  cover: {
+    height: 105,
+    width: 69,
+    maxHeight: '100%',
+    maxWidth: '100%'
+  },
+  container: {
+    flex: 1,
+    flexDirection: 'column',
+    height: '100%',
+    width: '100%',
+    maxHeight: 270,
+    maxWidth: 335,
```

Comment:
always beware when using maxHeight and Width because you are fixing de size for different mobile sizes

Reply:
ok... so, I can't use that in this way?

Reply:
nop